apps/backend/src/parse/dto/garms-transaction.dto.ts

```diff
@@ -90,7 +90,8 @@ export class GarmsTransactionDTO {
   @ValidateNested({ each: true })
   @Type(() => GarmsPaymentDTO)
   @IsArray()
-  @Validate(ArePaymentMethodsValid)
+  // TODO CCFPCM-534: Some files in existing dataset are invalid depending on how exchange rate is used
```

Comment:
I think these should be rejected as they may fail reconciliation unless we resolve?